app/models/office.rb

```diff
@@ -61,8 +59,12 @@ def field_order
     ]
   end
 
+  def telephone_number=(new_phone_number)
+    super cleanup_telephone_number(new_phone_number)
+  end
+
   def telephone_number
-    super.try { |x| x.gsub(' ', '') }
+    return format_telephone_number(cleanup_telephone_number(super))
```

Comment:
Am I right in saying that, if we do an up-front cleanup of existing telephone numbers, then the call to `cleanup_telephone_number` here is no longer required?